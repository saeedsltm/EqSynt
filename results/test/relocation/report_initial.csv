OT,Lat,LatErr,Lon,LonErr,Dep,DepErr,NST,RMS,GAP
2020-01-01 00:01:40.000000,35.972,-99.90,51.792,-99.90, 19.3,-99.90,  9, 0.0,-100
2020-01-01 00:03:20.000000,36.820,-99.90,50.932,-99.90, 19.2,-99.90,  5, 0.0,-100
2020-01-01 00:05:00.000000,35.579,-99.90,51.103,-99.90,  5.8,-99.90,  3, 0.0,-100
2020-01-01 00:06:40.000000,35.377,-99.90,52.251,-99.90,  6.3,-99.90,  6, 0.0,-100
2020-01-01 00:08:20.000000,35.545,-99.90,51.471,-99.90, 17.5,-99.90,  6, 0.0,-100
2020-01-01 00:10:00.000000,37.146,-99.90,52.276,-99.90, 16.0,-99.90,  7, 0.0,-100
2020-01-01 00:11:40.000000,35.441,-99.90,52.021,-99.90, 15.0,-99.90,  4, 0.0,-100
2020-01-01 00:13:20.000000,35.702,-99.90,52.270,-99.90,  9.6,-99.90,  8, 0.0,-100
2020-01-01 00:15:00.000000,36.588,-99.90,51.990,-99.90, 14.1,-99.90,  6, 0.0,-100
2020-01-01 00:16:40.000000,36.005,-99.90,51.626,-99.90, 14.1,-99.90, 10, 0.0,-100
