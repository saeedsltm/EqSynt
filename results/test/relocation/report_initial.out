 Year Date HRMM  Sec Latitud Error Longitud Error Depth Error NST  RMS GAP
 2020  1 1  0 1 40.0  35.972         51.792        19.3         9  0.0    
 2020  1 1  0 3 20.0  36.820         50.932        19.2         5  0.0    
 2020  1 1  0 5  0.0  35.579         51.103         5.8         3  0.0    
 2020  1 1  0 6 40.0  35.377         52.251         6.3         6  0.0    
 2020  1 1  0 8 20.0  35.545         51.471        17.5         6  0.0    
 2020  1 1  010  0.0  37.146         52.276        16.0         7  0.0    
 2020  1 1  011 40.0  35.441         52.021        15.0         4  0.0    
 2020  1 1  013 20.0  35.702         52.270         9.6         8  0.0    
 2020  1 1  015  0.0  36.588         51.990        14.1         6  0.0    
 2020  1 1  016 40.0  36.005         51.626        14.1        10  0.0    
